wagtail/snippets/views/snippets.py

```diff
@@ -858,6 +857,7 @@ def workflow_history_detail_view(self):
             object_icon=self.icon,
             header_icon="list-ul",
             workflow_history_url_name=self.get_url_name("workflow_history"),
+            _show_breadcrumbs=False,
```

Comment:
See https://static-wagtail-v5-2.netlify.app/admin/snippets/base/person/workflow_history/4/detail/3/ for why I added this, it's not ready to use breadcrumbs yet and I should've done this in 5.2